routers/routes/routes.go

```diff
@@ -749,6 +744,7 @@ func RegisterRoutes(m *macaron.Macaron) {
 				m.Post("/lock", reqRepoIssueWriter, bindIgnErr(auth.IssueLockForm{}), repo.LockIssue)
 				m.Post("/unlock", reqRepoIssueWriter, repo.UnlockIssue)
 				m.Get("/attachments", repo.GetIssueAttachments)
+				m.Get("/attachments/:uuid", repo.GetAttachment)
```

Comment:
You should be able to download attachments for archived repositories

Reply:
Good point, the existing GET above also needs to move out of this.

Reply:
Can you maybe post example code on how to resolve this? I'm not very familiar with how these routes work and my first idea was to duplicate the `Group` but that's probably not the best approach.

Reply:
Apparantly groups can duplicate according to existing code, I'll go with that.

Reply:
Fixed and verfied that archived issue and release attachments can be downloaded.